pkg/account/authorizer.go

```diff
@@ -226,14 +228,19 @@ func (a *authorizer) Authorize(ctx context.Context, tx solana.Transaction) (resu
 
 	cfg, err := a.config.Get(ctx, appIndex)
 	if err == app.ErrNotFound || cfg == nil || cfg.CreateAccountURL == nil {
+		if err == app.ErrNotFound {
```

Comment:
I recall you mentioned that if there is no app URL (where the wallet was created) we should throw an error. Does that mean you should only check `cfg.CreateAccountURL`? Why are you checking cfg?

Reply:
The `cfg.CreateAccountURL` is not in play here, we are throwing an error for apps that are un-configured.